src/richie/apps/courses/settings/__init__.py

```diff
@@ -47,6 +89,10 @@ class MyConfiguration(RichieCoursesConfigurationMixin, Configuration):
     ("richie/child_pages_list.html", _("List of child pages")),
     ("richie/homepage.html", _("Homepage")),
     ("richie/single_column.html", _("Single column")),
+    ("richie/three_columns_33.html", _("Three columns: 33/33/33%")),
+    ("richie/two_columns_50.html", _("Two columns: 50/50%")),
+    ("richie/two_columns_25_75.html", _("Two columns: 25% and 75%")),
+    ("richie/two_columns_75_25.html", _("Two columns: 75% and 25%")),
```

Comment:
Ok

Reply:
i don't really the one to use, the first is simpler but it will avoid to use any other 1/3 | 2/3, like a 30%|70% or a 40%|60%.

The second one is more explicit (i would tend to this one). 

@sampaccoud any advice ?

Reply:
+1 for percentage for the reason that you gave : it will be possible to define templates with different ratios...

Reply:
Well i will use the explicit one with the percentages, everyone will understand it as well as the fraction one.